kernel/csymtable.ml

```diff
@@ -193,13 +285,34 @@ and eval_to_patch env (buff,pl,fv) =
        See test-suite/primitive/float/coq_env_double_array.v
        for an actual instance. *)
```

Comment:
s/eval_tcode/coq_interprete/ in the above comment (since eval_tcode no longer exists)